src/Razor/src/Microsoft.CodeAnalysis.Remote.Razor/OOPTagHelperResolver.cs

```diff
@@ -75,17 +73,13 @@ public override async Task<TagHelperResolutionResult> GetTagHelpersAsync(Project
 
             try
             {
-                TagHelperResolutionResult result = null;
+                TagHelperResolutionResult? result = null;
                 if (factory != null)
                 {
                     result = await ResolveTagHelpersOutOfProcessAsync(factory, workspaceProject, projectSnapshot, cancellationToken).ConfigureAwait(false);
                 }
 
-                if (result is null)
-                {
-                    // Was unable to get tag helpers OOP, fallback to default behavior.
```

Comment:
Can you resurrect this comment please

Reply:
I just blindly run codefix to get it. Filled a roslyn issue for such case: https://github.com/dotnet/roslyn/issues/64051. Will fix by hands now